extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -471,9 +476,14 @@ public static Table readPartitionedTable(
                 StandaloneTableKey.getInstance(),
                 locationKeyFinder,
                 new ParquetTableLocationFactory(readInstructions),
-                null);
-        return new PartitionAwareSourceTable(tableDefinition, "Read multiple parquet files with " + locationKeyFinder,
-                RegionedTableComponentFactoryImpl.INSTANCE, locationProvider, null);
+                readInstructions.isRefreshing() ? TableDataRefreshService.getSharedRefreshService() : null);
+        return new PartitionAwareSourceTable(
+                tableDefinition,
+                readInstructions.isRefreshing() ? "Read refreshing parquet files with " + locationKeyFinder
+                        : "Read multiple parquet files with " + locationKeyFinder,
```

Comment:
Just a readability preference:
```suggestion
                readInstructions.isRefreshing()
                        ? "Read refreshing parquet files with " + locationKeyFinder
                        : "Read multiple parquet files with " + locationKeyFinder,
```

Reply:
Will do.